hug/interface.py

```diff
@@ -204,10 +204,8 @@ def __init__(self, route, function):
                     self.all_parameters.add(interface_name)
                 if internal_name in self.required:
                     self.required = tuple(
-                        [
-                            interface_name if param == internal_name else param
+                        interface_name if param == internal_name else param
                             for param in self.required
-                        ]
```

Comment:
One other note, because you removed a level of indentation the `for param` line should also be de-indented.

Reply:
Thank you for pointing this. I almost forgot this. Fixing this in a minute.